### DISCUSSION TITLE
希望脚本可以让哔哩哔哩重新支持在网页实现多p投稿

### DISCUSSION THREAD

**Discussion by Huaxidesu** [the1812/Bilibili-Evolved#3590] [功能建议 / Ideas]
新版哔哩哔哩不让分p投稿，因此必须使用第三方工具投稿，但是第三方工具只支持4g以下的视频投，必须先随便投两个视频，然后卡出网页bug出现添加视频才能继续投稿（在网页投稿不限制大小），这样做非常麻烦。既然还可以用这个功能并且能卡出添加视频，正说明这个功能只不过是在网页被屏蔽了。希望Bilibili-Evolved可以帮助各位up解决下这个问题。

**Comment by the1812** [comment]
不太熟投稿这块...
